Library/Homebrew/tab.rb

```diff
@@ -341,6 +341,20 @@ def write
     tabfile.atomic_write(to_json)
   end
 
+  # Helper method for checking truthiness of a value that may not be specified.
+  # `Tab.might_be?(:foo)` differs from `tab.foo` in that if `foo` is not
+  # specified `#might_be?` will return true.
+  def might_be?(key)
```

Comment:
Well, a tab can either have a key set, or it can not. If it's not specified, sometimes we want to be able to treat that as if the value was truthy, e.g.

```ruby
if tab.installed_on_request || tab.installed_on_request.nil?
```

In other words, do this if the tab was either installed on request, or if we don't have data for whether it was installed in request. That allows us to take actions only if we know for sure that a tab _wasn't_ installed on request.

Does that make sense? Any idea how I could improve the description or name of the method to get that idea across better while staying succinct?